src/vue/sheets/actor/character/TalentsTab.vue

```diff
@@ -88,6 +88,50 @@ async function upgradeTalent(talent: GenesysItem<TalentDataModel>) {
 	});
 }
 
+async function downgradeTalent(talent: GenesysItem<TalentDataModel>) {
+	if (talent.systemData.ranked === 'no') {
+		ui.notifications.info(game.i18n.format('Genesys.Notifications.TalentCannotBeDowngraded', { talentName: talent.name }));
+		return false;
+	}
+
+	const newEffectiveTier = talent.systemData.effectivePreviousTier;
+	const reward = talent.systemData.downgradeReward;
+
+	await toRaw(talent).update({
+		'system.rank': talent.systemData.rank - 1,
+	});
+
+	await toRaw(context.data.actor).update({
+		'system.experienceJournal.entries': [
+			...system.value.experienceJournal.entries,
+			{
+				amount: +reward,
+				type: EntryType.TalentRank,
+				data: {
+					name: talent.name,
+					id: talent.id,
+					tier: newEffectiveTier,
+					rank: talent.systemData.rank,
+				},
+			},
+		],
+	});
+}
```

Comment:
There is no consideration on this function, and the one below, to deal gracefully with the talent pyramid. Downgrading or deleting a talent might cause an invalid structure. Also see the comment below about new entries on the Journal.